internal/pkg/sqn/sqn.go

```diff
@@ -14,13 +14,15 @@ const UndefinedSeqNo = -1
 
 var DefaultSeqNo = []int64{UndefinedSeqNo}
 
-// Abstracts the array of document seq numbers
+// SeqNo Abstracts the array of document seq numbers
```

Comment:
nit:
"// SeqNo Abstracts the array of document seq numbers" -> "// SeqNo abstracts the array of document seq numbers"
seems like that was the convention so far